programs/ido-program/Anchor.toml

```
@@ -0,0 +1,15 @@
+[features]
+seeds = false
+skip-lint = false
+[programs.localnet]
+ido_program = "GYpxvUxtesyBSn69gnbfQChoUyJ7qdsG9nXS2Y2dQNH6"
+
+[registry]
+url = "https://api.apr.dev"
+
+[provider]
+cluster = "Localnet"
+wallet = "/Users/catrovacer/.config/solana/id.json"
```

Please make the wallet path generic otherwise it will only work for you

```suggestion
wallet = "~/.config/solana/id.json"
```